process/incubation.md

```diff
@@ -0,0 +1,138 @@
+* Name (must be unique within SSWG)
+* Description (what it does, why it is valuable, origin, and history)
+* Statement on alignment with SSWG mission
+* Preferred initial maturity level (see SSWG Graduation Criteria)
+* Sponsor (where applicable)
+* Initial committers (how long working on project)
+* Link to source (GitHub by default)
+* External dependencies (including licenses)
+* Release methodology and mechanics
+* License (Apache 2 by default)
+* Issue tracker (GitHub by default)
+* Communication channels (slack, irc, mailing lists)
+* Website (optional)
+* Social media accounts (optional)
+* Community size and any existing sponsorship
+
+The SSWG votes on pending proposals on a bi-weekly cadence, with the goal of voting on at least two proposals per month.
+
+### Graduation Criteria
+
+Every SSWG project has an associated maturity level: **Sandbox**, **Incubating**, or **Graduated**. Proposals should state their preferred initial maturity level, and the SSWG will take a vote to decide on the actual level. 
+
+A **supermajority** (two-thirds) is required for a project to be accepted as Incubating or Graduated. If there is not a supermajority of votes to enter at the Graduated level, then the votes toward Graduated are recounted as votes to enter at the Incubating level. If there is not a supermajority of votes to enter at the Incubating level, then all votes are recounted as **sponsorship** to enter at the Sandbox level. If there are not at least two sponsors, the Proposal is rejected.
+
+#### Sandbox Level
+
+To be accepted at the Sandbox level, a project must meet the [SSWG minimal requirements](#minimal-requirements) detailed below and be endorsed by at least two SSWG sponsors.
+
+Early adopters should treat early stage projects with extra care. While Sandbox projects are safe to try out, it is expected that some projects may fail and never move to the next maturity level. There is no guarantee of production readiness, users, or professional level support. As such, users must exercise their own judgment.
+
+#### Incubating Level
+
+To be accepted at Incubating level, a project must meet the Sandbox level requirements plus:
+
+* Document that it is being used successfully in production by at least three independent end users which, in the SSWG judgement, are of adequate quality and scope.
+* Have a healthy number of committers.
+* Demonstrate a substantial ongoing flow of commits and merged contributions.
+* Receive a supermajority vote from the SSWG to move to Incubation stage.
+
+#### Graduated Level
```

Comment:
One thing that isn't mentioned here is what is the responsibility the SSWG assumes when "graduating" a project.

For example, nothing is said about expectation about how security issues will be handled, or whether we require that projects will be guaranteed to work with the latest version of Swift ... or N versions of Swift?

Essentially, what is the ⭐️ aspect of a project being graduated, versus incubated?  Just the level of confidence in that project, which is fairly subjective?  Or is there something more?

Reply:
@tkremenek graduated projects are expected to follow the minimal requirements + best practices listed below

i changed the language there around addressing security issues and supporting swift and linux versions in https://github.com/swift-server/sswg/pull/1/commits/cd18097ee16045ec2a98afe5755bb4bec49eba32. wdyt?

Reply:
+1 on what Ted says. IMHO we should state that we will provide security updates to graduated projects in a timely manner in case the usual authors are on holiday or awol. 
Regarding Swift version support I’m unsure, clearly everything should support the latest released Swift version but I don’t know if we should require support for N releases back...

Reply:
@tomerd sorry, missed that, that sounds good!

Reply:
@tomerd https://github.com/swift-server/sswg/commit/cd18097ee16045ec2a98afe5755bb4bec49eba3 looks good to me too